README.md

```diff
@@ -18,51 +19,57 @@ sudo apt-get update;
+3) Insert token for bot into `appsettings.Development.json`
+3) Start a bot either from IDE (VS, VSC, Rider, ...) or command line.
+```
+dotnet run --project <path_to_src/Inkluzitron>
+```
 
-## Produkční deployment
+## Production deployment
 
 TBD
 
-## Struktura? Who knows
-
-- `src/` - Zdrojáky
-  - `Inkluzitron` - Adresář s projektem
-    - `bin/` - Adresář s binárkama
-    - `obj/` - Nepotřebujete vědět
-    - `Extensions/` - Rozšiřující metody, které vám mohou zjednodušit život, ale asi je nikdy nevyužijete.
-    - `Handlers/` - Třídy a metody pro zachytávání událostí. Nebudete potřebovat.
-    - `Modules/` - Zde budou třídy s moduly, které budou obsluhovat commandy události, reakce, ... *Většinu času budete implementovat zde.*
-    - `Services/` - Podpůrné služby, aby život byl krásnější. *Asi nikdy sem nebudete potřebovat.*
-    - `appsettings.json` - Hlavní konfigurace a současně šablona configu.
-    - `Inkluzitron.csproj` - Projektový soubor
-    - ... (Cokoliv dalšího, ptej se ostatních)
-  - `.editorconfig` - Nesahat
-  - `Inkluzitron.sln` - Spouštěcí soubor do projektu pro VS (možná bude fungovat i rider.)
-- `README.md` (Toto readme)
+## Structure
+
+- `src/` - Source codes
+  - `Inkluzitron` - Directory containing project
+    - `bin/` - Binaries
+    - `obj/` - *You don't need to know*
+    - `Extensions/` - Extending methods that can make your life easier
+    - `Handlers/` - Classes and methods for catching events
+    - `Modules/` - Classes and modules that will handle commands, events, reactions, ...
+    - `Services/` - Support services to make life nicer
+    - `appsettings.json` - Main config file and template for config
+    - `Inkluzitron.csproj` - Project file
+    - ... (For anything else just ask)
+  - `.editorconfig` - DO NOT TOUCH
+  - `Inkluzitron.sln` - Project file for VS (may work for Rider too)
+- `README.md` - The thing you are reading right now
+- `README.cs.md` - The thing you are reading right now but in Czech
 - `.gitignore`
 
-## Co je třeba vědět?
-
-- Cokoliv přidáte do configu, takto dejte do `appsettings.json`, aby ostatní věděli, co v tom vlastně je.
-- Všechno přidávejte formou PR. **NIKDO** nebude pushovat přímo do `master` branche.
-- Pokud nevíte. Ptejte se.
-- Dívejte se do konzole (to znamená stdout a stderr). Provádí se tam logování.
-- V projektu je využit dependency injection kontejner. Vyžaduje to knihovna Discord.NET.
-- Appka je napsaná, že by se mělo vše načítat dynamicky, tudíž, když budete chtít něco přidat, tak bude stačit udělat tyto jednoduché kroky:
-  - Vytvořte novou třídu v `Modules/`.
-  - Poděďte z bázové třídy `ModuleBase`.
-  - Užijte si zábavu. Pokud nevíte, tak se koukněte jinam.
+## What you need to know?
+
+- Anything you add to config also add to `appsettings.json` so that others know what is inside it.
+- Use PRs (Pull Requests) to add features or make changes. **NO ONE** can push directly to the `master` branch.
```

Comment:
There is strictly no need for a capitalised "Pull Requests" (and Chicago style would recommend against it), but there is nothing inherently wrong with this style. Just make sure to keep consistency in the future.